docs/writing-tests/accessibility-testing.md

```diff
@@ -6,29 +6,37 @@ Accessibility is the practice of making websites inclusive to all. That means su
 
 Accessibility tests audit the rendered DOM against a set of heuristics based on [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) rules and other industry-accepted best practices. They act as the first line of QA to catch blatant accessibility violations.
 
-Storybook’s official [a11y addon](https://storybook.js.org/addons/@storybook/addon-a11y) runs accessibility audits while you’re developing components to give you a fast feedback loop. It's powered by Deque's [axe-core](https://github.com/dequelabs/axe-core), which automatically catches up to [57% of WCAG issues](https://www.deque.com/blog/automated-testing-study-identifies-57-percent-of-digital-accessibility-issues/).
-
 ![Accessibility testing](./accessibility-testing-storybook.gif)
 
-## Setup a11y addon
+## Accessibility testing with a11y addon
+
+Storybook provides an official [a11y addon](https://storybook.js.org/addons/@storybook/addon-a11y). Powered by Deque's [axe-core](https://github.com/dequelabs/axe-core), which automatically catches up to [57% of WCAG issues](https://www.deque.com/blog/automated-testing-study-identifies-57-percent-of-digital-accessibility-issues/).
+
+### Set up the a11y addon
+
+If you want to run accessibility tests using the [addon](https://storybook.js.org/addons/@storybook/addon-a11y/), you'll need additional steps to set it up. Documented below is our recommendation.
 
-To enable accessibility testing with Storybook, you'll need to install the [`@storybook/addon-a11y`](https://storybook.js.org/addons/@storybook/addon-a11y/) addon. Run the following command:
+Run the following command to install the addon.
 
-```shell
-# With npm
-npm install @storybook/addon-a11y --save-dev
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/storybook-a11y-install.yarn.js.mdx',
+    'common/storybook-a11y-install.npm.js.mdx',
+  ]}
+/>
 
-# With yarn
-yarn add --dev @storybook/addon-a11y
-```
+<!-- prettier-ignore-end -->
 
-Update your Storybook configuration (in `.storybook/main.js`) to include the accessibility addon:
+Update your Storybook configuration (in `.storybook/main.js|ts`) to include the accessibility addon.
```

Comment:
Is it that common to have a `main.ts`? Doesn't seem worth the complication here.

Reply:
@kylegach yes, yes it is this was left here intentionally to avoid unwanted issues/questions about TS support as we had in the past, to which I responded and updated the documentation accordingly. The rest of the documentation will "fall in line" (pardon the bad pun) as well shortly